src/modelinstance.cpp

```diff
@@ -720,20 +708,20 @@ Status ModelInstance::loadOVCompiledModel(const ModelConfig& config) {
         SPDLOG_LOGGER_INFO(modelmanager_logger, "OVMS set plugin settings key: {}; value: {};", key, value.as<std::string>());
     }
 
-    const std::string supportedConfigKey = METRIC_KEY(SUPPORTED_CONFIG_KEYS);
-    std::vector<std::string> supportedConfigKeys;
+    auto prop = ov::supported_properties;
+    std::vector<ov::PropertyName> supportedConfigKeys;
     try {
-        std::vector<std::string> supportedConfigKeys2 = compiledModel->get_property(supportedConfigKey).as<std::vector<std::string>>();
+        std::vector<ov::PropertyName> supportedConfigKeys2 = compiledModel->get_property(prop);
         supportedConfigKeys = std::move(supportedConfigKeys2);
     } catch (std::exception& e) {
-        SPDLOG_LOGGER_DEBUG(modelmanager_logger, "Exception thrown from IE when requesting target device: {}, CompiledModel metric key: {}; Error: {}", targetDevice, supportedConfigKey, e.what());
+        SPDLOG_LOGGER_DEBUG(modelmanager_logger, "Exception thrown from IE when requesting target device: {}, CompiledModel metric key: {}; Error: {}", targetDevice, prop.name(), e.what());
         return StatusCode::OK;
     } catch (...) {
-        SPDLOG_LOGGER_DEBUG(modelmanager_logger, "Exception thrown from IE when requesting target device: {}, CompiledModel metric key: {}", targetDevice, supportedConfigKey);
+        SPDLOG_LOGGER_DEBUG(modelmanager_logger, "Exception thrown from IE when requesting target device: {}, CompiledModel metric key: {}", targetDevice, prop.name());
         return StatusCode::OK;
     }
     SPDLOG_LOGGER_DEBUG(modelmanager_logger, "Logging model:{}; version: {};target device: {}; CompiledModel configuration", getName(), getVersion(), targetDevice);
-    for (auto& key : supportedConfigKeys) {
+    for (ov::PropertyName& key : supportedConfigKeys) {
```

Comment:
Please check this PR review done by @atobiszei : https://github.com/openvinotoolkit/model_server/pull/1514
His comments suggested to use `auto` wherever possible (instead of `ov::PropertyName` and `vector<ov::PropertyName>`). I have done that in my commit: https://github.com/openvinotoolkit/model_server/pull/1514/commits/92d72748bc616a8be57a149c5dd34a7d4c6b8660